jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -118,26 +118,37 @@ blockchain_configuration:
   # Halving formula: constant * ratio ^ (#epoch after epoch_start / epoch_rate)
   #
   reward_parameters:
-    halving: # or use "linear" for the linear formula
+    linear: # or use "linear" for the linear formula
       # In the linear formula, it represents the starting point of the contribution
       # at #epoch=0, whereas in halving formula is used as starting constant for
       # the calculation.
-      constant: 100
+      constant: 3835616440000
```

Comment:
I don't understand why you needed to change this value?

Reply:
This is the constant amount that will be returned per epoch, as agreed in September

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.

Reply:
See above.